src/agent/rustjail/src/mount.rs

```diff
@@ -288,6 +289,35 @@ pub fn init_rootfs(
         }
     }
 
+    if expose_configfs {
+        if let Some(annotations) = spec.annotations() {
+            annotations.iter().for_each(|(k, v)| {
+                if k == "io.katacontainers.pkg.oci.container_type" && v != "pod_sandbox" {
+                    let err = mount(
+                        Some("configfs"),
+                        format!("{}/sys/kernel/config", rootfs).as_str(),
+                        Some("configfs"),
+                        MsFlags::MS_NODEV
+                            | MsFlags::MS_NOSUID
+                            | MsFlags::MS_NOEXEC
+                            | MsFlags::MS_RELATIME,
+                        None::<&str>,
+                    );
+                    match err {
+                        Ok(_) => (),
```

Comment:
What's about using if let Err(x) = {...} to simplify the error handling ?

Reply:
done, thanks!